CHANGELOG.md

```diff
@@ -11,12 +11,14 @@ created.
 ## Upcoming
 
 ### Added
+- New extensions of SegmentationModelBases `HeadAndNeckBase` and `ProstateBase`.
```

Comment:
Add maybe "Use these classes to build your own Head&Neck or Prostate models, by just providing a list of foreground classes"